cmd/tools/rpcwrappers/main.go

```diff
@@ -96,6 +97,7 @@ var (
 		"client.history.StreamWorkflowReplicationMessages":          true,
 		"metricsClient.history.StreamWorkflowReplicationMessages":   true,
 		"retryableClient.history.StreamWorkflowReplicationMessages": true,
+		"client.admin.HealthCheck":                                  true,
```

Comment:
why does this have to be excluded? the generated one looks fine

Reply:
removed. use the generated one.